### PR TITLE
Add BugSense to the app

### PR DESCRIPTION
**Pull request by karllindmark**
Add BugSense to the app via our all time favorite activity called BaseIntelActivity. That should do.

I've created a new project on my previous account (where BF3 Droid was), but I'm still waiting for an OSS upgrade to be able to "share" it with others. :+1: 

I checked to see that it was actually working by adding a `throw new NullPointerException("Hello Peter!!!");` into the `onCreate(Bundle)` method, and it did register it was an issue. Huzzah. :+1: 

screen shot 2014-02-14 at 00 36 11

@peter-budo 

Build on CI:
http://ci.peterscorner.co.uk/job/BF4-Intel-PR/27/console

### CODE REVIEW

**Comment by peter-budo** [comment]
@karllindmark 
Setup BF4 Intel on Bugsense and please share access

File: BF4Intel/build.gradle [hunk 1]
```diff
@@ -38,5 +41,6 @@ dependencies {
         'com.squareup:otto:1.3.4',
         'com.google.code.gson:gson:2.2.4',
         'com.squareup.picasso:picasso:2.1.1',
-        'nl.qbusict:cupboard:1.0.4'
+        'nl.qbusict:cupboard:1.0.4',
+        'com.bugsense.trace:bugsense:3.5'
```
Comment (peter-budo) [inline BF4Intel/build.gradle]: > Your first step is to copy the bugsense-3.6.zip

https://www.bugsense.com/docs/android

So 3.5 is previous version

**Comment by karllindmark** [comment]
@peter-budo 
Like I said earlier - "I've created a new project on my previous account (where BF3 Droid was), but I'm still waiting for an OSS upgrade to be able to "share" it with others." - it seems as if the basic account doesn't allow 'team members/collaborators". So I'm waiting to hear back. :+1:

File: BF4Intel/build.gradle (hunk 1 above)
Comment (karllindmark) [inline BF4Intel/build.gradle]: Fixed. I just took the version number from their Gradle setup page, humbug. :hatching_chick:

**Comment by karllindmark** [comment]
@peter-budo: Alright, you should have access to BF3 Droid, BF4 Intel as well as Battle Chat on BugSense (created a team). Could you verify?

**Comment by peter-budo** [comment]
@karllindmark no I do not see it and now even old BF3 Droid disappeared...

**Comment by karllindmark** [comment]
What, really? That's even weirder... Can you "accept" a team in some way?

**Comment by karllindmark** [comment]
@peter-budo: OK, try again now. The old BF3 Droid "project" is removed, as we never used it. :+1:

**Comment by peter-budo** [comment]
@karllindmark I'm in. I see Bf4 Intel and Battle-Chat but not BF3 Droid (guess that does not matter)

PS: You should setup Bugsense to sent reports to PivoTalTracker instead of Github

**Comment by karllindmark** [comment]
@peter-budo: Cool. I set up Pivotal now, should create stories automatically if I understood it correctly. :-)

**Pull request merged**